### DISCUSSION TITLE
Custom page, print table

### DISCUSSION THREAD

**Discussion by Causeofhell** [filamentphp/filament#13062] [Help]
### Package

Other

### Package Version

v3.2

### How can we help you?

I need a bit of help, i have a custom page in which i have a form and below the form i want to print a table filled with information contained in one array i came with this idea
` public function submit()
    {
        $calculator = new LoanCalculator(
            $this->apply_amount,
            $this->first_payment_date,
            $this->interest_rate,
            $this->term,
            TermPeriod::from($this->term_period),
            $this->late_payment_penalties
        );

        $this->results = $calculator->calculate(InterestType::from($this->interest_type));
    }`
    then in the view 
    `<x-filament::page>
    <form wire:submit.prevent="submit">
        {{ $this->form }}

        <div class="mt-4">
            <x-filament::button type="submit">
                Calculate
            </x-filament::button>
        </div>
    </form>

    @if ($results)
        <div class="mt-8">
            <table class="min-w-full divide-y divide-gray-200">
                <thead>
                    <tr>
                        <th scope="col" class="px-6 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider">Date</th>
                        <th scope="col" class="px-6 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider">Amount to Pay</th>
                        <th scope="col" class="px-6 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider">Penalty</th>
                        <th scope="col" class="px-6 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider">Principal Amount</th>
                        <th scope="col" class="px-6 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider">Interest</th>
                        <th scope="col" class="px-6 py-3 text-left text-xs font-medium text-gray-500 uppercase tracking-wider">Balance</th>
                    </tr>
                </thead>
                <tbody>
                    @foreach ($results as $result)
                        <tr>
                            <td class="px-6 py-4 whitespace-nowrap">{{ $result['date'] }}</td>
                            <td class="px-6 py-4 whitespace-nowrap">{{ $result['amount_to_pay'] }}</td>
                            <td class="px-6 py-4 whitespace-nowrap">{{ $result['penalty'] }}</td>
                            <td class="px-6 py-4 whitespace-nowrap">{{ $result['principal_amount'] }}</td>
                            <td class="px-6 py-4 whitespace-nowrap">{{ $result['interest'] }}</td>
                            <td class="px-6 py-4 whitespace-nowrap">{{ $result['balance'] }}</td>
                        </tr>
                    @endforeach
                </tbody>
            </table>
        </div>
    @endif
</x-filament::page>
`
what i want is to print the filament table

**Comment by griggsk** [comment]
You can do this by defining your table in a different custom page.

Define your filament table in a new custom page, ResultTable: https://filamentphp.com/docs/3.x/tables/adding-a-table-to-a-livewire-component.
Get the $results to use in your table in the mount method:
`public function mount $results)
    {
        $this->results = $results;
    }`
Create the result-table view
` {{ $this->table}}`

Replace the table in your form page with  <livewire:result-table :result="$results"/>
